contracts/token-registrars/StandardizedTokenRegistrar.sol

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { SafeTokenTransfer } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/libs/SafeTransfer.sol';
+import { Upgradable } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/upgradable/Upgradable.sol';
+
+import { IInterchainTokenService } from '../interfaces/IInterchainTokenService.sol';
+import { IStandardizedTokenRegistrar } from '../interfaces/IStandardizedTokenRegistrar.sol';
+import { ITokenManagerType } from '../interfaces/ITokenManagerType.sol';
+import { ITokenManager } from '../interfaces/ITokenManager.sol';
+import { IStandardizedToken } from '../interfaces/IStandardizedToken.sol';
+
+import { Multicall } from '../utils/Multicall.sol';
+
+import { AddressBytesUtils } from '../libraries/AddressBytesUtils.sol';
+
+contract StandardizedTokenRegistrar is IStandardizedTokenRegistrar, ITokenManagerType, Multicall, Upgradable {
```

Comment:
Can we combine the two registrars into one? Easier to manage operations around it and for UI to integrate